lib/tests/streamlit/server_test.py

```diff
@@ -550,6 +550,37 @@ def test_rotates_one_port(
                 )
 
 
+class UnixSocketTest(unittest.TestCase):
+    """Tests start_listening uses a unix socket when socket.address starts with
+       unix:// """
+
+    def get_httpserver(self):
+        httpserver = mock.MagicMock()
+
+        httpserver.add_socket = mock.Mock()
+
+        return httpserver
+
+    def test_unix_socket(self):
+        app = mock.MagicMock()
+
+        os.environ["HOME"] = "/home/koen"
```

Comment:
Is there a way this can be mocked instead of actually changing the environment variables, which may have unexpected effects on the testing system?

Also -- believe it or not, we had a 10-minute engineering discussion about this line here.   **Would you mind changing this to something that's more obviously fake**?

It's a code-readability thing -- when someone is scanning the tests, it's easier to see that "/home/superfakehomedir" is meant to be test data, and not a piece of real data left in by mistake.

Relatedly, we'd like you to know that we're going to start adding Contributors to our Release Notes, so your GitHub username will be forever commemorated in the release that your fix or enhancement was added into.  <3

Reply:
You are totally right about the superfakehomedir. Changed it. Also fixed the patch of environ variable. Thanks for the feedback!

Adding the Contributors to the Release Notes is great! Maybe I should pimp my profile :-)